src/Repository/MagazineRepository.php

```diff
@@ -524,6 +524,7 @@ public function findRemoteForUpdate(): array
             ->andWhere('m.apDomain IS NULL')
             ->andWhere('m.apDeletedAt IS NULL')
             ->andWhere('m.apTimeoutAt IS NULL')
+            ->addOrderBy('m.apFetchedAt', 'ASC')
```

Comment:
What is this change for?

Reply:
tbh this may be entirely unnecessary because I have no sample results, but this function is used in actor update command to batch update actors that meet those criteria, and I figured that in such case the oldest fetched actors should get queued first

could remove that if it isn't needed

Reply:
I mean it doesn't hurt, I just wanted to know what your reason was :)